modules/nuage/manifests/vrs.pp

```diff
@@ -63,6 +69,14 @@
     }
   }
 
+  if $enable_hw_offload {
+    vs_config { 'other_config:hw-offload':
+      value  => 'true',
+      notify => Service['openvswitch'],
```

Comment:
Wouldn't we be notifying Service[$nuage::params::nuage_vrs_service] like the other parameters?

Reply:
Good point.

Reply:
Good catch, thanks, I will fix it.

Reply:
Done